analyzer/src/main/kotlin/managers/PubCacheReader.kt

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2017-2022 HERE Europe B.V.
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer.managers
```

Comment:
I believe this should be moved to `package org.ossreviewtoolkit.analyzer.managers.utils` instead.

Reply:
I was undecided between `utils` and `managers`. The `*DependencyHandler` classes are in the `managers` package as well. Would you agree the `Handler` classes should be moved to `utils` too?

Reply:
Yes, I always wanted to move them, but forgot about it, so I did it now: https://github.com/oss-review-toolkit/ort/pull/5086